src/index.html

```diff
@@ -798,9 +851,8 @@ <h2 class="controls-title">Duration</h2>
         // data for https://www.exhibit.so/api/exhibits/Zmtdg3cwREXzNGCuHgmm
         // const annotationsJson =
         //   "http://localhost:3000/api/exhibits/dIVPgGD9ostBUU1ckIZZ";
-
-        const annotationsJson =
-          "http://192.168.1.233:3000/api/exhibits/dIVPgGD9ostBUU1ckIZZ";
+        // const annotationsJson =

```

Comment:
This variable is referenced in the `initStory()` function, which itself is not commented out (though the call to it is). Should we comment out all of initStory to prevent confusion? Do we need to retain all of this at all?